docs/reference/advanced_configuration.rst

```diff
@@ -306,12 +306,12 @@ If you want to use the Tab Menu in a different way, you can replace the Menu Tem
 Translations
 ^^^^^^^^^^^^
 
-The translation parameters and domain can be customised by using the
-``translation_domain`` and ``translation_parameters`` keys of the extra array
+The label translation parameters and domain can be customised by using the
+``label_translation_parameters`` and ``label_catalogue`` keys of the extra array
 of data associated with the item, respectively::
 
     $menuItem->setExtras([
-        'translation_parameters' => ['myparam' => 'myvalue'],
+        'label_translation_parameters' => ['myparam' => 'myvalue'],
         'translation_domain' => 'My domain',
```

Comment:
label_catalogue

Reply:
Thanks, and what about other doc files? Should we replace all of them?
![image](https://user-images.githubusercontent.com/30590523/82193472-b63bc000-98f5-11ea-89f2-a623a573aa69.png)

Reply:
A lot of them are valid. I think.
It works for formMapper, showMapper, etc... The main issue was for group/menu.

It would stay like this.